src/Service/UpdaterWorker.php

```diff
@@ -270,7 +271,9 @@ public function process(Job $job, SignalHandler $signal): array
         }
 
         if ($packageName === FriendsOfPhpSecurityAdvisoriesSource::SECURITY_PACKAGE) {
-            $this->scheduler->scheduleSecurityAdvisory(FriendsOfPhpSecurityAdvisoriesSource::SOURCE_NAME);
+            $this->scheduler->scheduleSecurityAdvisory(FriendsOfPhpSecurityAdvisoriesSource::SOURCE_NAME, $id);
+        } elseif (preg_match('{[@/]github.com[:/]([^/]+/[^/]+?)(\.git)?$}i', $package->getRepository())) {
+            $this->scheduler->scheduleSecurityAdvisory(GitHubSecurityAdvisoriesSource::SOURCE_NAME, $id);
```

Comment:
Problem with calling this at the very end of the package update is that it won't be called if any issue is detected during the package update e.g. an invalid composer.json in a single branch. I think this is less of an issue with the friendsofphp package because we can kind of assume that their composer.json is always ok. But might have to move all of this up a bit